src/points.rkt

```diff
@@ -192,7 +194,8 @@
   (eval-errors baseline newpcontext))
 
 (define (errors-score e)
-  (let-values ([(reals unreals) (partition ordinary-value? e)])
+  (define repr (get-representation 'binary64))
+  (let-values ([(reals unreals) (partition (curryr ordinary-value? repr) e)])
```

Comment:
I think there should be a comment noting that `e` contains error measurements (in bits) which are always floats. But on second thought they should never be anything but ordinary values (as in I think that's not possible), so we should be able to significantly simplify this function.